src/test/java/net/minecraftforge/debug/FluidMixTest.java

```diff
@@ -0,0 +1,25 @@
+package net.minecraftforge.debug;
+
+import net.minecraft.block.BlockStone;
+import net.minecraft.init.Blocks;
+import net.minecraft.world.biome.Biome;
+import net.minecraft.world.biome.BiomeHills;
+import net.minecraftforge.event.world.BlockEvent;
+import net.minecraftforge.fml.common.Mod;
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent;
+
+@Mod(modid = "fluidmixtest", name = "FluidMixTest", version = "0.0.0")
```

Comment:
Needs `acceptableRemoteVersions = "*"` in the `@Mod` annotation, otherwise it'll prevent vanilla clients connecting to Forge servers launched from a dev env for working on Forge, which makes testing vanilla-Forge compatiblity more difficult.